### ISSUE TITLE
[Bug report]: Error Type: Callback Error getLedgerDeviceStatus

### ISSUE THREAD

**Issue by LionelCutz** [iotaledger/firefly#2907]
### Platform

Desktop

### Version

1.4.2

### Description

Don´t know if this is a new issue, but in Version 1.4.2. it is still the same. When i open IOTA legacy app on ledger and start searching. I still get stuck here:

Error in Firefly:

App Version: 1.4.2

platform: macOS (High Sierra 10.13.6.)
platformVersion: 10.13.6
platformArchitecture: x64
cpuCount: 8
totalMem: 10240.0 MB
freeMem: 4147.1 MB

Error Type: Callback Error getLedgerDeviceStatus

TypeError: Cannot read property 'selectSeed' of undefined
    at o (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2882882)
    at s (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232324)
    at onSuccess (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232138)
    at e.<computed>.<computed> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2204288)
    at Function.<anonymous> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2197511)

### Expected behaviour

Migration trinity to firefly

### Actual behaviour

error as described. 

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

steps to migrate with ledger

### Operating System

Mac

### Diagnostics

```shell
App Version: 1.4.2

platform: macOS
platformVersion: 10.13.6
platformArchitecture: x64
cpuCount: 8
totalMem: 10240.0 MB
freeMem: 4147.1 MB

Error Type: Callback Error getLedgerDeviceStatus

TypeError: Cannot read property 'selectSeed' of undefined
    at o (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2882882)
    at s (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232324)
    at onSuccess (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232138)
    at e.<computed>.<computed> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2204288)
    at Function.<anonymous> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2197511)
```

### Developer Profile

No

### Profile Type

Hardware (Ledger)

### Crash Status

Infinite loading symbol

### Error Messages

App Version: 1.4.2

platform: macOS
platformVersion: 10.13.6
platformArchitecture: x64
cpuCount: 8
totalMem: 10240.0 MB
freeMem: 4147.1 MB

Error Type: Callback Error getLedgerDeviceStatus

TypeError: Cannot read property 'selectSeed' of undefined
    at o (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2882882)
    at s (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232324)
    at onSuccess (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232138)
    at e.<computed>.<computed> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2204288)
    at Function.<anonymous> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2197511)

### Error Logs

```shell
App Version: 1.4.2

platform: macOS
platformVersion: 10.13.6
platformArchitecture: x64
cpuCount: 8
totalMem: 10240.0 MB
freeMem: 4147.1 MB

Error Type: Callback Error getLedgerDeviceStatus

TypeError: Cannot read property 'selectSeed' of undefined
    at o (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2882882)
    at s (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232324)
    at onSuccess (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2232138)
    at e.<computed>.<computed> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2204288)
    at Function.<anonymous> (file:///Applications/Firefly.app/Contents/Resources/app.asar/public/build/index.js:2:2197511)
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by rajivshah3** [comment]
@nicole-obrien is this a duplicate of https://github.com/iotaledger/firefly/issues/2783?

**Comment by nicole-obrien** [comment]
Yeah this is duplicate and fixed in 1.5.0 releasing Tuesday.
